pkg/interface/src/views/landscape/components/ChannelPopoverRoutes/Notifications.tsx

```diff
@@ -25,7 +25,7 @@ export function ChannelNotifications(props: ChannelNotificationsProps) {
   };
 
   return (
-    <Col mb="6" gapY="4">
+    <Col mb="6" gapY="4" display="block">
```

Comment:
`Col`'s whole function is that it's shorthand for `display: flex; flex-direction: column` — is there a reason `flexShrink={0}` won't suffice here?

Reply:
It did suffice. Changed it.